.github/workflows/ci.yml

```
@@ -2,167 +2,132 @@ name: CI
 
 on:
   push:
-    branches: [ main, codex/implement-phase-1-of-issue-#4 ]
+    branches: [ main ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)